src/main/java/org/opensearch/performanceanalyzer/reader/ClusterDetailsEventProcessor.java

```diff
@@ -226,18 +229,20 @@ public String getRole() {
             return role;
         }
 
-        public boolean getIsMasterNode() {
-            // TODO : this is added to support backward compatibility for _cat/master fix.
+        public boolean getIsClusterManagerNode() {
+            // TODO : this is added to support backward compatibility for _cat/cluster_manager fix.
             //  We can remove this later if the writer changes has been adapted by all clusters in
             // fleet.
-            // query cat master api directly to read the node's role in case the ClusterDetailsEvent
-            // received from writer does not have "IS_MASTER_NODE" field.
-            if (isMasterNode == null) {
-                final String electedMasterHostAddress =
-                        RcaControllerHelper.getElectedMasterHostAddress();
-                isMasterNode = this.hostAddress.equalsIgnoreCase(electedMasterHostAddress);
+            // query cat cluster_manager api directly to read the node's role in case the
+            // ClusterDetailsEvent
+            // received from writer does not have "IS_CLUSTER_MANAGER_NODE" field.
+            if (isClusterManager == null) {
+                final String electedClusterManagerHostAddress =
+                        RcaControllerHelper.getElectedClusterManagerHostAddress();
+                isClusterManager =
+                        this.hostAddress.equalsIgnoreCase(electedClusterManagerHostAddress);
```

Comment:
nit : can be a single line
```suggestion
                isClusterManager = this.hostAddress.equalsIgnoreCase(electedClusterManagerHostAddress);
```